static/legacy/README.md

```diff
@@ -1,17 +0,0 @@
-# DO NOT REMOVE THIS DIRECTORY
```

Comment:
Did you read this message?

Reply:
Well, I'm aware of it, but it is a legacy directory, and it is available on the git tree. We can even make a permanent tree with a branch or a tag.

cc @joesepi @Trott what do you folks think?

Reply:
@nschonni We can ask the TSC for create a repo where we archive a lot of things about nodejs. for example this content

Reply:
That is another alternative to be honest. I don't see a reason of keeping a legacy folder just because someone many years ago wrote this...

Reply:
Did you read the contents of this file, not the one line? These files are used/hotlinked by legacy docs

Reply:
If they're manually linked on NGINX it doesn't mean they could live statically somewhere else.

I just don't see how these static legacy files should be part of the website repository.